src/components/Tooltip.tsx

```diff
@@ -1,85 +1,112 @@
-import React, { ReactNode, useState } from "react"
-import { Box, useColorModeValue } from "@chakra-ui/react"
+import React, { ReactNode } from "react"
```

Comment:
Unfortunately, this is not going to be the desired result in the creation of the tooltip.

The primary focus with creating or recreating these components is to utilize what Chakra UI provides to simplify the need for building custom logic. There is a [Tooltip](https://chakra-ui.com/docs/components/tooltip/usage) component available, which means at least one of two things will need to occur:

---
First and foremost is creating a tooltip theme config in `src / @chakra-ui / gatsby-plugin / components` to handle the custom styling set by the Design System. Check out the `component.utils.ts` file in this sub-directory and the other component theme files to see how the default component theming from Chakra is applied.

Secondly, if the tooltip needs additional reusable logic that is not already built into Chakra's `Tooltip`, then the component file used is simply a wrapper of the same name with Chakra's component imported as `ChakraTooltip` and rendered accordingly.

```tsx
import { Tooltip as ChakraTooltip } from '@chakra-ui/react'

const Tooltip = (props) => {
  // Any custom logic here

  /* 
   * Can be a simple return like below, 
   * unless further structure needs to be created for the custom logic
   */ 
  return <ChakraTooltip />
}

export default Tooltip
```

---
If no additional reusable logic is needed, then no component file is needed. This means we only import the Tooltip straight from the Chakra package any time it's needed and have the styling isolated in the theming config. In this case, a storybook story would still need to be made, but this new file will reside in `src / components / BaseStories`.

Have you tried any of the above? Let me know if you are running into trouble and I can try to assist. 🙂

Reply:
`Tooltip` also don't allow to focus on Tooltip content using keyboard

Reply:
Thank you for review @TylerAPfledderer

I tried the `Tooltip` component from Chakra but in DS, the tooltip is shown as not opened on focus that's why I used it because if I want to add some custom logic to the `Tooltip` component then all the Chakra logic for tooltip is overridden by it. If `Tooltip` is going to opened on focus then I think I can use Chakra Tooltip without custom logic.

Another catch with Chakra `Tooltip` is that it uses `Portal` which makes the hard to navigate using Keyboard.

Reply:
Hi @TylerAPfledderer, I have created the **Tooltip** theme config as per your request and will be working on the story shortly. However, I'm facing an issue with setting the `z-index` of the tooltip content despite using `PortalManager`. Do you have any suggestions on how to resolve this?

Reply:
![IMG_20230830_194956](https://github.com/ethereum/ethereum-org-website/assets/91947037/189c0780-84dc-46e6-ba81-95c06a4cb4d4)

The z-index of tooltip content is higher than the navbar. in the main site, tooltip is hidden from the navbar.

Reply:
@amit-ksh this is actually "expected" behavior of this component... and by expected, I mean that if you checked out the [Placement section of the Chakra Docs](https://chakra-ui.com/docs/components/tooltip/usage#placement) and checked this with the "top" button in the example, you will see this exact same thing!

For now, I would ignore this behavior and I will make a note to check in with the Chakra core team to check if this is intended.